src/cli/Microsoft.DotNet.xunit.performance.runner.cli/project.json

```diff
@@ -20,7 +20,8 @@
     "Microsoft.DotNet.xunit.performance.run.core": "99.99.99-dev",
     "Microsoft.NETCore.DotNetHostPolicy":"1.0.1-rc2-002357-00",
     "NETStandard.Library": "1.5.0-rc2-23931",
-    "xunit.extensibility.execution": "2.1.0"
+    "Microsoft.xunit.extensibility.execution.netcore": "1.0.0-prerelease",
```

Comment:
This was a repackaged version of the same fixed version of the dependency so that we are also able to restore for netstandard1.3. 

From https://github.com/karajas/buildtools/blob/eef26e8e6e46c4b11b721ce84c11bdd02fcc9207/src/nuget/Microsoft.xunit.extensibility.execution.netcore.nuspec ,
<file src="Microsoft.xunit.extensibility.execution.netcore\xunit.execution.dotnet.dll" target="lib/netstandard1.0" />